src/test/java/tests/TyposTests.java

```diff
@@ -0,0 +1,24 @@
+package tests;
+
+import constants.Urls;
+import org.openqa.selenium.By;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class TyposTests extends BaseTest {
+    @Test
+    public void checkboxOneCanBeCheckedTest() {
+        driver.get(Urls.TYPOS_URL);
+
+        //check line #1 of paragraph
+        String lineOneText =
+                driver.findElement(By.cssSelector("div[class='example']:nth-child(1) > p")).getText();
+        Assert.assertEquals(lineOneText, "This example demonstrates a typo being introduced. It does it randomly on each page load.");
```

Comment:
Please move validation to the end of test, because code after it won't be executed if assertion happens